PigLatin/PigLatin.cs

```diff
@@ -81,7 +81,7 @@ public static bool tests()
                 TranslateWord("dye") == "yeday" && 
                 TranslateWord("bystander") == "ystanderbay" &&
                 TranslateWord("yellow") == "ellowyay" &&
-                TranslateWord("tsktsk") == "tsktskay";
+                TranslateWord("tsktsk") == "tsktskay";//
         }
```

Comment:
Once test I see missing is a test for a word that starts with a vowel and ends with a `y`.

While not in the assignment, another case you could have checked for is words that end in y and start with a vowel. In cases such as those, "agency" for example, you end up with a double "y" (e.g. agencyyay).